src/LanguageServer/Impl/Microsoft.Python.LanguageServer.ruleset

```diff
@@ -0,0 +1,92 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we can make one global ruleset in `src` folder. @MikhailArkhipov , what do you think?

Reply:
Makes sense

Reply:
I think some of these messages may actually be useful in some cases, so I'll update this with the ruleset move and change these to informational instead of completely disabled.